src/resources/sorting_network_master/sortingnetwork.py

```diff
@@ -0,0 +1,241 @@
+
```

Comment:
Why'd you remove the license?

Reply:
It's still a license

Reply:
The "LICENSE" file still exists though

Reply:
Also, I think it may be better to rework the shell commands to not rely on the usage of `-c`?

Reply:
I agree, but I couldn't find another way to do it using the `BufferedInputStream` from the `SortingNetworkFetcher`.

Reply:
Gonna look into it, probably not gonna find anything either but whatever

Reply:
The only other way I can think of is to extract the script from the JAR before running it, but that doesn't feel right...

Reply:
I found a way!

Reply:
ooo lemme hear (unresolving for now just to mark that it may be possible)

Reply:
I didn't find a way